src/training/train.cpp

```diff
@@ -157,27 +171,85 @@ void TestInference(RecordInput* input, GraphInference* inference) {
 
 }
 
-void Train(RecordInput* input, GraphInference* inference) {
-  inference->SSVMInit(FLAGS_regularization_const, FLAGS_svm_margin);
+void Train(RecordInput* input, GraphInference* inference, int fold_id) {
```

Comment:
Are you saying that I should have one common initialisation that initialise all the various parameters, and not two different for the two trainings?